webapp/src/main/java/life/qbic/AppConfig.java

```diff
@@ -42,4 +44,13 @@ public UserRepository userRepository(UserDataStorage userDataStorage) {
     return UserRepository.getInstance(userDataStorage);
   }
 
+  /**
+   * Creates the event store instance
+   * @return 1.0.0
+   */
+  @Bean
+  public EventStore eventStore() {
+    return new EventStore(new TemporaryEventRepository());
```

Comment:
I prefere a Singleton vs constructor, to also guarantee that we only have on instance of an event store in the application instance.

```suggestion
    return EventStore.instance(new TemporaryEventRepository());
```